feature/interface/aggregate/ate_tests/balancing_test/balancing_test.go

```diff
@@ -203,6 +203,13 @@ func (tc *testCase) clearAggregateMembers(t *testing.T) {
 	}
 }
 
+func (tc *testCase) waitForLACP(t *testing.T) {
```

Comment:
Rename this to verifyDUT?

Also, the `tc.verifyDUT` below on line 417 looks like a misnomer. Rename that to `tc.getCounters`?